### DISCUSSION TITLE
[@types/react-imgix] - `SrcSetParams` properties should be optional

### DISCUSSION THREAD

**Discussion by parkersweb** [DefinitelyTyped/DefinitelyTyped#61504] [Issues with a @types package] [answered]
At the moment the interface for the `srcSetOptions` prop is defined as:
```
interface SrcSetParams {
    widths: number[];
    widthTolerance: number;
    minWidth: number;
    maxWidth: number;
}
```
However I think all properties of this interface are optional? If the prop isn't supplied react-imgix uses the default values.

[a comment by an automated account is not shown]

**Comment by sherwinski** [comment, marked as answer]
Hey @parkersweb, you're correct in that each of these properties are optional. I'll make an update to that today.

**Reply by sherwinski** [reply]
PR for it here https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61511

**Comment by parkersweb** [comment]
Ah - awesome - thanks so much @sherwinski !

**Reply by sherwinski** [reply]
No problem - updates just got merged in

**Comment by parkersweb** [comment]
Perfect - all good now - thanks 🙂
